dts/arm/nuvoton/npcx7.dtsi

```diff
@@ -717,6 +734,13 @@
 			label = "TACH_2";
 			status = "disabled";
 		};
+
+		psl_out: psl_out0 {
```

Comment:
psl_out: psl-out0

or maybe leave off the 0 ?

phandles use _
properties use -

Reply:
Done. I will follow this in the following CLs.